src/content/github/file/utils/dropdown.tsx

```diff
@@ -17,15 +17,34 @@ export async function createDropdown({
   previousElement: HTMLElement;
   selectedOptions: string[];
 }) {
-  const editButton = document
-    .querySelector('[data-testid="more-edit-button"]')!
-    .closest("div")!;
-  const dropdownButton = editButton.cloneNode(true) as HTMLElement;
-  const textNode: HTMLElement = dropdownButton.querySelector('[data-component="IconButton"]')!;
+  // Build the button out of the Raw/copy/download button group
```

Comment:
If you're looking at a file at a commit, the old button we were building from does not exist. I adjusted the code to use a button that exists on both types of view (the raw/copy/download group). Additionally, I made the code a bit safer by removing `!`s